ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/cv_pathfinder_regions.schema.yml

```diff
@@ -31,6 +33,8 @@ keys:
           description: |-
             All sites are placed in a default zone "<region_name>-ZONE" with ID 1.
           primary_key: name
+          unique_keys:
+            - id
```

Comment:
This will not enforce uniqueness across regions. I don't think we need that for id, but I think we need name to be unique across all regions. That should be added as another unique key on the top level `sites.name`

Reply:
name uniqueness is handled in code.